ui/app/controllers/access-control/policies/policy.js

```diff
@@ -11,7 +11,7 @@ import { tracked } from '@glimmer/tracking';
 import { alias } from '@ember/object/computed';
 import { task } from 'ember-concurrency';
 
-export default class PoliciesPolicyController extends Controller {
+export default class AccessControlPoliciesPolicyController extends Controller {
```

Comment:
Before this PR, there was already a /policies route in the app. You'll see a few files like this, where things are moved into the new /access-control/policies section instead.